Fix Capitals in navigation bar

CallumScar opened the pull request:
Fixes Category Acronyms Capitals

CallumScar committed:
Fix Capitals in navigation bar

ajdlinux committed:
Add new 'sideblock' element to sidebar

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>

ajdlinux committed:
Merge branch 'master' of github.com:sthbrx/pelican-octopress-theme

ajdlinux committed:
Attempt to add author names and times on the front page

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>

ajdlinux committed:
Make author name and post date appear on index

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>

duilio commented:
it seems like a very long MR to just fix the category acronyms capitals... also I don't get what was the bug here? The problem is when you have an acronym in a category?

daxtens committed:
indent lists!

Signed-off-by: Daniel Axtens <dja@axtens.net>

daxtens committed:
move analytics to head where google wants it

Signed-off-by: Daniel Axtens <dja@axtens.net>

daxtens committed:
only include disqus content on article pages

Signed-off-by: Daniel Axtens <dja@axtens.net>

daxtens committed:
Consolidate JS usage

 - drop modernizr, flash video support etc.
 - drop ender.js
 - rewrite things that required ender to use vanilla js
 - drop other un-necessary JS
 - move the tiny amount of JS into the main page.

The result is that everything is fast.

Signed-off-by: Daniel Axtens <dja@axtens.net>

daxtens committed:
drop old ie compat tags

Signed-off-by: Daniel Axtens <dja@axtens.net>

ajdlinux committed:
Display byline on article view page

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>

ajdlinux committed:
Move article metadata to top of post

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>

ajdlinux committed:
Display full author list on posts

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>

ajdlinux committed:
Add authors links to sidebar

Signed-off-by: Andrew Donnellan <andrew.donnellan@au1.ibm.com>
